content/tutorials/build-apps/build-a-desktop-wallet-in-javascript.md

```diff
@@ -0,0 +1,1433 @@
+                <button type="submit">Confirm</button>
+            </div>
+        </form>
+    </dialog>
+```
+
+6. In `view/renderer.js` add at the top:
+
+```javascript
+window.electronAPI.onOpenSeedDialog((_event) => {
+    const seedDialog = document.getElementById('seed-dialog');
+    const seedInput = seedDialog.querySelector('input');
+    const submitButton = seedDialog.querySelector('button[type="submit"]');
+
+    submitButton.addEventListener('click', () => {
+        const seed = seedInput.value;
+        window.electronAPI.onEnterSeed(seed)
+        seedDialog.close()
+    });
+
+    seedDialog.showModal()
+})
+
+window.electronAPI.onOpenPasswordDialog((_event) => {
+    const passwordDialog = document.getElementById('password-dialog');
+    const passwordInput = passwordDialog.querySelector('input');
+    const submitButton = passwordDialog.querySelector('button[type="submit"]');
+
+    submitButton.addEventListener('click', () => {
+        const password = passwordInput.value;
+        window.electronAPI.onEnterPassword(password)
+        passwordDialog.close()
+    });
+
+    passwordDialog.showModal()
+});
+```
+
+Start up the application:
+
+```console
+npm run start
+```
+
+On first run, It should first prompt you for an account seed and then for a password. After you have created a wallet this way, you should close the application and start it up a second time: On second run it should prompt you for the password, and you should see the same result as in the last step.
+
+To run the reference application found in `content/_code-samples/build-a-wallet/desktop-js` for this step, run:
+
+```console
+npm run password
+```
+
+### 6. Styling
+
+**Full code for this step:**
+[`6_styling.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/6_styling.js).
+[`library/3_helper.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/library/3_helper.js).
+[`library/4_helper.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/library/4_helper.js).
+[`library/5_helper.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/library/5_helper.js).
+[`view/6_preload.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/view/6_tx-preload.js).
+[`view/6_styling.html`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/view/6_styling.html).
+[`view/6_renderer.js`]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/desktop-js/view/6_tx-renderer.js).
+
+After finishing this step the application should look like this:
+
+![Screenshot: Step 6, style application with css](img/javascript-wallet-6.png)
+
+1. Copy the folder `bootstrap` and its contents to your project directory. 
```

Comment:
What's the link to `bootstrap`?

Reply:
Copying a whole folder is hard in a browser tutorial because it requires using Git to download. Instead I'd recommend using the same "copy + paste" with filenames instructions you've been using so far. `bootstrap` only has two files, even though they're long, so it should be straightforward. (Although I would want to double check that the file scrolled right instead of displaying vertically, because that'd be really bad formatting otherwise.

Reply:
Bumping this issue